kerchunk/_grib_idx.py

```diff
@@ -0,0 +1,250 @@
+import ujson
```

Comment:
Since this module does not have user functions (which should be exposed via `kerchunk.grib`), I think the whole module ought to have a docstring to explain this.